vta/python/vta/top/graphpack.py

```diff
@@ -246,32 +246,42 @@ def visit_call(self, call):
 
 class BT(Exception):
     pass
-def get_subgraph(expr, start_name, stop_name):
+def get_subgraph(expr, start_name, stop_name, start_name_indx=0, stop_name_indx=0):
     """ We assume stop_name only appears once for simplicity.
         This constraint will be lifted in the future.
         bitpack_start and bitpack_end are both inclusive.
     """
     bitpack_start = op.op.get('annotation.bitpack_start')
     bitpack_end = op.op.get('annotation.bitpack_end')
     anf = run_opt_pass(expr, transform.ToANormalForm())
-    def _recursion(anf, start_found, stop_found):
+    start_name_num = 0
+    stop_name_num = 0
+    def _recursion(anf, start_found, stop_found, start_name_num, stop_name_num):
```

Comment:
CHECK: when start_name==stopname, start index should be smaller than stop index.